source/_dashboards/gauge.markdown

```diff
@@ -150,23 +153,94 @@ segments:
   - from: 40
     color: '#43a047'
   - from: 60
-    color: '#ffa600'
+    color: '#4396a0'
   - from: 65
+    color: '#437ba0'
+  - from: 70
+    color: '#4362a0'
+```
+
+#### Outdoor temperature
+
+<p class='img'>
+<img src='/images/dashboards/gauge_segments_feelslike_temperature.png' alt='Screenshot of a gauge card with colored segments for temperature.'>
+Screenshot of a gauge card with colored segments for temperature.
+</p>
+
+```yaml
+type: gauge
+entity: sensor.feelslike_temperature
+needle: true
+min: -15
+max: 50
+segments:
+  - from: -15
+    color: '#8438e0'
```

Comment:
Please use double quotes for strings. For more information, checkout out our developer documentation on YAML standards:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

PS: This comment applies to other places in this PR too